crates/cast/tests/cli/main.rs

```diff
@@ -299,6 +299,42 @@ Address:     0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
 Private key: 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 
 
+"#]]);
```

Comment:
Would you mind adjusting this example to include `--accounts 3`?

Reply:
Addressed by 97c0226008c69872c5112f543b9e032cdfd1fe4e